phpBB/config/default/container/services_files.yml

```diff
@@ -1,4 +1,35 @@
 services:
+    attachment.delete:
```

Comment:
i moved the previous attachment management tasks (so `files.*`) services under `services_attachment.yml` for quick fix, though that's might not the best solution, so do whatever you fancy, and i'll rebase.